RNSound/RNSound.m

```diff
@@ -173,17 +174,22 @@ - (NSDictionary *)constantsToExport {
     }
 
     if (category) {
-        if (mixWithOthers) {
+        if (mixWithOthers && duckOthers == false) {
```

Comment:
Please update the code structure as follows, thanks.

```
if (mixWithOthers) {
  if (duckOthers) {
  ...
  } else {
  ...
  }
} else {
...
}
```

Reply:
code fixed